src/gui/providers/ogr/qgsgeopackageitemguiprovider.cpp

```diff
@@ -344,11 +355,48 @@ void QgsGeoPackageItemGuiProvider::vacuumGeoPackageDbAction( const QString &path
 }
 
 void QgsGeoPackageItemGuiProvider::createDatabase( const QPointer< QgsGeoPackageRootItem > &item )
+{
+  if ( item )
+  {
+    const QgsSettings settings;
+    const QString lastUsedDir = settings.value( QStringLiteral( "UI/lastGeoPackageeDir" ), QDir::homePath() ).toString();
```

Comment:
Where's this actually set? 
```suggestion
    const QString lastUsedDir = settings.value( QStringLiteral( "UI/lastGeoPackageDir" ), QDir::homePath() ).toString();
```

Reply:
Fixed, thanks